code/game/machinery/modular_shield.dm

```diff
@@ -490,6 +498,13 @@
 
 	if(connected_node)
 
+		//checks if the node allows boosters and if we are a booster
+		if((connected_node.allow_boosters == FALSE) && (booster == TRUE)) {
```

Comment:
putting aside the {} (shouldn't be here), you don't need to wrap these == checks

Reply:
more then that you could write this as 
```suggestion
		if(!connected_node.allow_boosters && booster)
```
and it would read just fine